typescript/infra/config/environments/mainnet3/agent.ts

```diff
@@ -412,7 +407,7 @@ const hyperlane: RootAgentConfig = {
     rpcConsensusType: RpcConsensusType.Fallback,
     docker: {
       repo,
-      tag: '9c056c7-20240911-154357',
+      tag: 'ed77033-20241008-222921',
```

Comment:
you'll have to rebase + obtain a new image build after https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4613 is merged